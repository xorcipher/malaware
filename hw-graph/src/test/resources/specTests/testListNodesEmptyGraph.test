# Tests that the getNodes method works properly for a graph with no nodes.

# Return all the nodes of a graph in String format.

CreateGraph graph

ListNodes graph
