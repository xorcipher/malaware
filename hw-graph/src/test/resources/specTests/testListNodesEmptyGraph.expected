# Tests that the getNodes method works properly for a graph with no nodes.

# Return all the nodes of a graph in String format.

created graph graph

graph graph contains:
